java/client/src/main/java/glide/api/commands/TransactionsCommands.java

```diff
@@ -0,0 +1,42 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands for the "Transactions Commands" group for a standalone client.
+ *
+ * @see <a href="https://redis.io/commands/?group=transactions">Transactions Commands</a>
+ */
+public interface TransactionsCommands {
+    /**
+     * Marks the given keys to be watched for conditional execution of a transaction. Transactions
+     * will only execute commands if the watched keys are not modified before execution of the
+     * transaction.
+     *
```

Comment:
Add a cross slot notice, and a corresponding test.

Reply:
Added the notice but I thought the corresponding test is just using `{}` in the key? https://github.com/Bit-Quill/glide-for-redis/blob/70389e830ab3998377bf7f1e3850286f4f0830fc/java/integTest/src/test/java/glide/SharedCommandTests.java#L4583-L4586

Let me know if this is not enough and maybe what you meant.

Reply:
https://github.com/Bit-Quill/glide-for-redis/blob/462feb549e89ccb82dd9278a9868e81c033524d3/java/integTest/src/test/java/glide/cluster/CommandTests.java#L657